src/app/examples/alert-example/alert-example.component.ts

```diff
@@ -2,25 +2,41 @@ import { Component, ViewContainerRef } from '@angular/core';
 
 import { ModalController } from '@kirbydesign/designsystem/modal';
 import { AlertConfig } from '@kirbydesign/designsystem/modal';
+import { of } from 'rxjs';
```

Comment:
The linter is complaining about these imports. They need to be at top as they are 3. party.

Reply:
Well, that was embarrassing... fixed!

Reply:
In regards to the "2 second delay", I wanted to show the "cheapest possible solution" to having a RxJs Observable do something.... granted.... I could simply make it wait 10ms... that way the impact wouldn't be annoying.